### DISCUSSION TITLE
Unsupported PageScrollMode.CONTINUOUS in Android and iOS SDKs

### DISCUSSION THREAD

**Discussion by vivekpshaligram** [readium/kotlin-toolkit#561] [General]
The PageScrollMode.CONTINUOUS setting, which allows for uninterrupted scrolling through content without page breaks, is not available in the Readium SDK implementations for Android and iOS. This limitation means that e-reading applications built using Readium SDK on these platforms cannot provide a continuous scrolling experience for EPUB books.

**Comment by mickael-menu** [comment]
Which `PageScrollMode.CONTINUOUS` setting are you referring to?

But yes, we don't support continuous scrolling in EPUB publications.

**Comment by vivekpshaligram** [comment]
Which PageScrollMode.CONTINUOUS setting are you referring to? - Yes

Should we include this feature in the next release, or is there an option to implement continuous scroll for both Android and iOS?

**Reply by mickael-menu** [reply]
No, we have no plan to support this at the moment.
